ruff.toml

```diff
@@ -0,0 +1,41 @@
+target-version = "py39"
```

Comment:
Can you add something to the readme how to use this file?

Reply:
This file is a configuration file for [ruff](https://github.com/astral-sh/ruff) which is an extremely fast linter for Python. It is automatically called by pre-commit (right before one commits), which is included in the README.

Documentation for the rules in this file is at https://beta.ruff.rs/docs/rules/

Reply:
If we want to change the lint standard from the used pylint to ruff we should change the lint run on the ci as well. We should make sure that this is done on every PR.

Reply:
Good idea, I was not able to run the CI on my own earlier, but I can give it a try.

To be clear, ruff currently doesn't fully replace Pylint in general, but for this project's needs it might. You seem to be using a pretty permissive pylint configuration.